test/test_masks.py

```diff
@@ -0,0 +1,19 @@
+import sys
+sys.path.append('masks/mask_class')
```

Comment:
when you make it part of the `lensless` package you won't have to do this and can import as e.g.:
```
from lensless.mask import CodedAperture, PhaseContour, FresnelZoneAperture
```